src/hooks/__internal__/useCharacterCount/useCharacterCount.test.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from "react";
+import { render, screen } from "@testing-library/react";
+import { act } from "react-dom/test-utils";
```

Comment:
**suggestion(non-breaking)**: what do you think of using RTL's wrapper for `act`? [The maintainers recommend using it mainly for consistency reasons](https://github.com/testing-library/testing-library-docs/issues/524#issuecomment-658681951)